examples/stm32f4_discovery/display/nokia_5110/project.cfg

```diff
@@ -0,0 +1,27 @@
+
+[general]
+name = n5110
+
+[build]
+device = stm32f407vg
+clock = 168000000
+buildpath = ${xpccpath}/build/stm32f4_discovery/display/${name}
+
+[program]
+tool = openocd
+
+[defines]
+CORTEX_VECTORS_RAM = 0
```

Comment:
I should have been clearer, sorry.
The define is deprecated, the parameter is used instead.

The Blinky example have the vector table relocated in RAM, so we can test it. Maybe that's not a good idea though, since it will be copied around.

You can leave in the commit that removes this, I will make a separate example for relocating the vector table.